### PR TITLE
mailing: replace mailchimp tags with mailtrain tags

### PR DESCRIPTION
**Pull request by harunurhan**
* INSPIR-2721

### CODE REVIEW

File: backend/inspirehep/mailing/templates/mailing/jobs/weekly/base.html [hunk 1]
```diff
@@ -889,7 +889,7 @@
                                                                 style="padding: 0px 18px 9px; text-align: center;"
                                                                 valign="top">
 
-                                                                <a href="*|ARCHIVE|*" target="_blank">View this email in
+                                                                <a href="[LINK_BROWSER]" target="_blank">View this email in
```
Comment (michamos) [inline backend/inspirehep/mailing/templates/mailing/jobs/weekly/base.html]: this is probably not needed, it gets added by mailtrain at the bottom

File: backend/inspirehep/mailing/templates/mailing/jobs/weekly/base.html [hunk 2]
```diff
@@ -1294,15 +1294,10 @@
...
-                                                                <br>
                                                                 Want to change how you receive these emails?<br>
-                                                                You can <a href="*|UPDATE_PROFILE|*">update your
-                                                                    preferences</a> or <a href="*|UNSUB|*">unsubscribe
-                                                                    from this list</a>.<br>
-                                                                <br>
-                                                                *|IF:REWARDS|* *|HTML:REWARDS|* *|END:IF|*
+                                                                You can <a href="[LINK_PREFERENCES]">update your
+                                                                    preferences</a> or <a href="[LINK_UNSUBSCRIBE]">unsubscribe
+                                                                    from this list</a>.
```
Comment (michamos) [inline backend/inspirehep/mailing/templates/mailing/jobs/weekly/base.html]: same

**Comment by harunurhan** [comment]
We will use those ones in our template and remove the `mailtrain` ones, since these fit better with the current template. (I am 90% sure, `MERGE_TAGS` work with RSS campaign too)

**Comment by michamos** [comment]
do you think they work recursively?

**Comment by harunurhan** [comment]
as far as I checked, `feedchecker` just passes the html content to the campaign.. I hope that later `mailtrain` replaces the tags as usual.

**Comment by michamos** [comment]
I mean not sure `mailtrain` will replace merge tags recursively (because the `RSS_ENTRY` is already a merge tag).

**Comment by harunurhan** [comment]
there seems to be no easy way to validate this without deploying though :/

**Pull request merged**